### ISSUE TITLE
Felicity PETG filament swelling/melting ABOVE the nozzle

### ISSUE THREAD

**Issue by AndyH0** [DoESLiverpool/somebody-should#1649]
SHORT STORY: Having problems with the blue PETG swelling and melting ABOVE the nozzle and then the filament will not advance or retract without pulling things apart to rectify. Happened twice. I don't think it's a nozzle blockage issue - good flow from nozzle between this problem. I have left Felicity with the filament ready to reload and hopefully it works for next user. Ran out of time to try again.

LONG STORY: First attempt: Good flow from nozzle but filament did not stick to existing blue masking tape. Aborted. The bed was levelled using some lined note paper from the bin. Thicker than normal printer paper. Second attempt: no flow from nozzle during warmup. Aborted. Tried feeding filament. Nothing emerging like a blocked nozzle. But noticed a bulge of blue PETG above the nozzle. By loosening the four screws to drop the nozzle, I was able to retract the filament and snip off the bulge and eventually retrieve the melted clump from the top of the nozzle. The filament was withdrawn. The filament was nearly worn through where the feeder has skidded so snipped off the filament and reloaded. Still no filament emerged, but a whisker of stranded wire goes easily into the nozzle. So nozzle seems to be clear. The filament wasn't aligned. The filament end was bent and I had snipped the filament at an angle that didn't help. The end of the filament was straightened and re-snipped. The filament loaded fine and a good flow emerged from nozzle. Tried my print again. Third attempt: The first layer outline was fine, a 3 mm internal hole outline was fine, but a 6 mm hole outline got dragged out of position so had to abort the print again. Fourth attempt: no flow from the nozzle during warmup. Above the nozzle the filament looks thin and melted. Tried to retract the filament. It came out of the nozzle but had a melted section and a blob on the end that jammed in the bowden tube. By removing the bowden tube and advancing the filament little by little and flexing the bowden tube I was able to work the lump out of the bowden tube. I have left Felicity with the filament prepared but unloaded but ready to go and I don't think the nozzle is blocked so fingers crossed it works for next user.

**Comment by AndyH0** [comment]
Tried to use Felicity again today. I had similar problems to last time with filament not sticking to bed and then on next attempt no flow from nozzle at all. Rather than try to feed material to the nozzle, I retracted the material, trimmed, and tried again. Retracting the material straight away I think avoided bigger problems of the material swelling up above the nozzle. It seemed to repeat this cycle of not sticking to the bed, then no flow, reload material, flow restore. Each time I tried levelling the bed on different thicknesses of paper. 0.40 mm was too thick. 0.17 mm thick paper, and 50% speed, finally the filament stayed stuck on the blue tape and print was nice if slow at the 50% speed. Didn't want to jinx it by trying speeding it up!
